nengo/utils/numpy.py

```diff
@@ -138,7 +138,7 @@ def norm(x, axis=None, keepdims=False):
         If True, the reduced axes are left in the result. See `np.sum` in
         newer versions of Numpy (>= 1.7).
     """
-    return np.sqrt(np.sum(x**2, axis=axis, keepdims=keepdims))
```

Comment:
Why not cast it up here, too?

Reply:
We could (there are probably other places we could do the same as well).  But also the only reason I was touching these functions was because I was getting type errors in the `rmse` part of the `allclose` fixture, which is no longer part of this PR.  So we could just revert this commit as well.